### DISCUSSION TITLE
Stop using SENSOR

### DISCUSSION THREAD

**Discussion by HenrikAerobase** [OpenRadioss/OpenRadioss#3037] [Q&A]
Hello, 

I would like to stop an analysis using a sensor which measures the distance between two nodes. I have the following input for the sensor and the stop, respectively.

STARTER: 
```
/SENSOR/DIST/1
Nodal distance sensor
# Time delay
    0.0001
#   Node_1   Node_2                 Dmin                Dmax
     36561   200000                0.005               0.037 
```

ENGINE:
```
/STOP/LSENSOR
         1

#
```
The Starter runs without error, when the Engine i launched I receive the following error:
```
MESSAGE ID :         73


 ** ERROR IN SOLVER INPUT DECK CARD: STOP 
```

I am running the latest release (Sept 13).  Any suggestion?

Henrik

**Comment by PaulAltair** [comment]
I can replicate, in engine file it is mandatory to have the option line populated (even if the values are 'default'), e.g. below should work

/STOP/LSENSOR
1          
         0         0         0         0         0         0         0

**Comment by HenrikAerobase** [comment]
Thanks Paul, it works perfectly!
